src/Swarm/TUI/View.hs

```diff
@@ -321,7 +321,7 @@ maybeScroll :: (Ord n, Show n) => n -> Widget n -> Widget n
 maybeScroll vpName contents =
   Widget Greedy Greedy $ do
     ctx <- getContext
-    result <- withReaderT (availHeightL .~ 1000000) (render contents)
+    result <- withReaderT (availHeightL .~ 10000) (render contents)
```

Comment:
What's the max message queue size?  Could it ever reach 10000 lines?

Reply:
Hah, it could be 10 001 because of the padding maybe? Also, I guess line overflow and multiline messages can make it arbitrarily long. I will test it.

Reply:
It does not work anyway, so I filed #629, let's fix that one separately. :disappointed: